packages/itwinui-react/src/core/Buttons/SplitButton/SplitButton.tsx

```diff
@@ -79,11 +89,14 @@ export const SplitButton = React.forwardRef((props, forwardedRef) => {
       ref={ref}
     >
       <Button
+        as='button'
         styleType={styleType}
         size={size}
         onClick={onClick}
         ref={forwardedRef}
         {...rest}
```

Comment:
if we're adding `buttonProps`, then `rest` props can be moved to the parent (breaking change)

alternatively, we can remove `buttonProps` and add `wrapperProps` (similar to checkbox and radio).

will wait for @gretanausedaite's opinion

Reply:
I choose to have wrapperProps.

Reply:
If I'm adding wrapperProps to the <Box> parent, what should do I do with its `title` and `ref` properties?

Reply:
- `className`, `style`, `title` and `ref` should go on the main button.
- `buttonProps` should be removed.
- `wrapperProps` should be added and spread over the wrapper.

Reply:
Added

Reply:
these `iui-button-split` and `iui-disabled` classes do not belong on the Button.
<img width="383" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/95e7dd21-c196-4aff-afa5-7ba2662de643">

Reply:
Fixed

Reply:
also remove `className` from the wrapper. it should only be applied on the Button.

i will also paste my comment from earlier again:
> `className`, `style`, `title` and `ref` should go on the main button. (you don't even need to destructure the first three, as they will be handled by `...rest`. only `ref` needs to be added manually)

Reply:
Done

Reply:
not done
<img width="311" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/d7cec9a7-b4e6-4a61-be17-c115fc4b6e6f">

Reply:
My mistake, I mixed this up with SideNavigation. I'll fix the changes I pushed to that, and then get back to this.

Reply:
`className` has been removed from `wrapper`